### PR TITLE
Fix Docs and test

### PR DESCRIPTION
**Pull request by seanjyjy**
**Pr Overview**
* Fix Test formats
* Fix JSdocs
* Update Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ tool. Check out our app [here!](https://algosort.netlify.app/)
 
 <br/>
 
-<p align="center"><strong>Radix Sort preview</strong></p>
+<p align="center"><strong>Bucket Sort preview</strong></p>
 
-<img src="images/RadixSort.gif" alt="" />
+<img src="images/BucketSort.gif" alt="" />
```
Comment (keanecjy) [inline README.md:31]: The bucket sort gif looks like it is using the old colors before my PR was merged? Should update to reflect the new one.

File: src/__tests__/visualizer/algorithm/sortingalgorithms/mergeSort.test.js [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
 
 test('Already Sorted with duplicates', () => {
-  const arr = [
-    { id: 1, height: 3, isShift: false, xDirection: 0, pos: 0, prevPos: 0 },
-    { id: 2, height: 3, isShift: false, xDirection: 10, pos: 1, prevPos: 1 },
-    { id: 3, height: 3, isShift: false, xDirection: 20, pos: 2, prevPos: 2 },
-    { id: 4, height: 3, isShift: false, xDirection: 30, pos: 3, prevPos: 3 },
-  ];
-  const actual = mergeSort(arr);
   const expected = [
```
Comment (keanecjy) [inline src/__tests__/visualizer/algorithm/sortingalgorithms/mergeSort.test.js]: The expected array for duplicates case and ascending case looks the same? Might want to create a global array for both to use.

File: src/visualizer/sortingvisualizer/component/selectors/algorithmselector/AlgorithmSelector.js [hunk 3]
```diff
@@ -69,6 +77,11 @@ const AlgorithmChooser = () => {
     }
   };
 
+  /**
+   * A drop down list which displays a list of algorithms available in Sort-Algo.
+   *
+   * @type {JSX.Element} A drop down list of algorithms available.
```
Comment (keanecjy) [inline src/visualizer/sortingvisualizer/component/selectors/algorithmselector/AlgorithmSelector.js]: There shouldn't be any description after type.

File: src/contactus/body/Form.js [hunk 4]
```diff
@@ -37,7 +38,7 @@ const Form = () => {
   /**
    * A drop down list to provide a list of types of message to choose from.
    *
-   * @type {JSX.Element}
+   * @type {JSX.Element} A drop down list.
```
Comment (keanecjy) [inline src/contactus/body/Form.js]: There shouldn't be any description after type.

File: src/contactus/body/Form.js [hunk 5]
```diff
@@ -37,7 +38,7 @@ const Form = () => {
...
    */
   const menu = (
```
Comment (keanecjy) [inline src/contactus/body/Form.js]: I think this menu should be an arrow function instead?

**Review by keanecjy — CHANGES_REQUESTED** [review]
Some nits to fix.

File: .gradle/buildOutputCleanup/cache.properties [hunk 6]
```diff
@@ -0,0 +1,2 @@
+#Sun Dec 20 18:14:12 SGT 2020
+gradle.version=6.5
```
Comment (keanecjy) [inline .gradle/buildOutputCleanup/cache.properties]: Can u also delete those gradle files?

**Review by keanecjy — APPROVED** [review]
LGTM! Great work.

File: src/visualizer/sortingvisualizer/component/selectors/algorithmselector/AlgorithmSelector.js [hunk 7]
```diff
@@ -34,6 +34,9 @@ const AlgorithmChooser = () => {
 
   const [algorithm, setAlgorithm] = useState('Bubble Sort');
 
+  /**
+   * List of available Algorithms available in Sort-Algo
```
Comment (Ashley-Lau) [inline src/visualizer/sortingvisualizer/component/selectors/algorithmselector/AlgorithmSelector.js:38]: Fullstop

File: src/visualizer/sortingvisualizer/component/legend/LegendHeader.js [hunk 8]
```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import './styles.css';
 
+/**
+ * Header for the Legend
```
Comment (Ashley-Lau) [inline src/visualizer/sortingvisualizer/component/legend/LegendHeader.js:5]: Fullstop missing

File: src/contactus/footer/Footer.js [hunk 9]
```diff
@@ -4,7 +4,10 @@ import './styles.css';
 import { MdCopyright } from 'react-icons/md';
 
 /**
- * Footer at the end of page.
+ * Footer at the end of page
```
Comment (Ashley-Lau) [inline src/contactus/footer/Footer.js:7]: FullStop missing

**Review by Ashley-Lau — CHANGES_REQUESTED** [review]
A few knit picks in docs.

**Pull request merged**